### PR TITLE
Merge conflicts fixed

### PR DESCRIPTION
**Pull request by dakshgautam**

### CODE REVIEW

File: squealy/squealy-web/src/Components/ViewModeResults.js [hunk 1]
```diff
@@ -66,6 +67,16 @@ export default class ViewOnlyResults extends Component {
     setUrlParams(payloadObj.params)
   }
 
+  //Update the type of chart selected
+  updateChartType = (value) => {
+    let payloadObj = JSON.parse(JSON.stringify(this.state.payloadObj))
+    payloadObj.params['chartType'] = value
+    this.setState({
+      payloadObj: payloadObj,
```
Comment (swapnil95) [inline squealy/squealy-web/src/Components/ViewModeResults.js]: Chart type should be saved in a single place

File: squealy/squealy-web/src/Components/ViewModeResults.js [hunk 2]
```diff
@@ -66,6 +70,15 @@ export default class ViewOnlyResults extends Component {
...
+    }, () => this.updateUrl())
```
Comment (swapnil95) [inline squealy/squealy-web/src/Components/ViewModeResults.js]: Replace () => this.updateUrl() with this.updateUrl. Only providing the reference of the function will do the job

File: squealy/squealy-web/src/Components/ViewModeResults.js [hunk 3]
```diff
@@ -28,13 +28,18 @@ export default class ViewOnlyResults extends Component {
...
     } else {
       payloadObj = formatTestParameters(propsData.chart.parameters, 'name', 'default_value')
     }
-    postApiRequest(DOMAIN_NAME+'squealy/'+ propsData.chart.url+'/', payloadObj,
-        this.onSuccessTest, this.onErrorTest, 'table')
-    this.setState({payloadObj: payloadObj}, this.updateUrl)
+    postApiRequest(DOMAIN_NAME + 'squealy/' + propsData.chart.url + '/', payloadObj,
+      this.onSuccessTest, this.onErrorTest, 'table')
+    payloadObj.params['chartType'] = propsData.chart.type
+    const getUrlParameters = getUrlParams()
```
Comment (swapnil95) [inline squealy/squealy-web/src/Components/ViewModeResults.js]: Change the variable name from getUrlParameters to urlPrameters

File: squealy/squealy-web/src/Components/ViewModeResults.js [hunk 4]
```diff
@@ -80,37 +93,48 @@ export default class ViewOnlyResults extends Component {
...
+          }
+
+        </div>
+        <div className="chart-type-select">
+          {this.state.payloadObj.params &&
+            <SquealyDropdown
+              name='chartType'
+              options={GOOGLE_CHART_TYPE_OPTIONS}
+              selectedValue={this.state.payloadObj.params['chartType']}
+              //onChangeHandler={(value) => selectedChartChangeHandler('type', value,null,null,2)}
```
Comment (hemny-singh) [inline squealy/squealy-web/src/Components/ViewModeResults.js]: Remove commented code.

File: squealy/squealy-web/src/Components/ParamDefinitionModal.js [hunk 5]
```diff
@@ -201,8 +200,8 @@ export default class ParamDefinitionModal extends Component {
             <div className='col-md-8'>
               <input type='text' name='testValue' 
                 value={this.state.paramDefinition.test_value}
-                onChange={(e) => this.onChangeParamHandler('test_value', e.target.value)}
-                onBlur={() => this.validateParamValueFormat('test_value', 'errorTestValue')}/>
+		onChange={(e) => this.onChangeParamHandler('test_value', e.target.value,'errorTestValue')}
```
Comment (hemny-singh) [inline squealy/squealy-web/src/Components/ParamDefinitionModal.js]: Add space after ,

File: squealy/squealy-web/src/Components/ParamDefinitionModal.js [hunk 6]
```diff
@@ -101,10 +101,10 @@ export default class ParamDefinitionModal extends Component {
     this.setState({ selectedType: value, paramDefinition: currentParamDefinition })
   }
 
-  onChangeParamHandler = (key, value) => {
+  onChangeParamHandler = (key, value,errorField) => {
```
Comment (hemny-singh) [inline squealy/squealy-web/src/Components/ParamDefinitionModal.js]: Add space after comma

**Pull request merged**